File: android/app/src/main/java/com/sumitrack/android/domain/models/Product.kt
package com.sumitrack.android.domain.models

import java.math.BigDecimal
import java.time.Instant

data class Product(
    val id: String,
    val fkTenant: String,
    val name: String,
    val price: BigDecimal,
    val taxRate: BigDecimal,
    val isActive: Boolean,
    val createdAt: Instant,
    val updatedAt: Instant,
    val syncStatus: SyncStatus,
)

File: backend/src/Sumitrack.Api/Models/Entities/Product.cs
namespace Sumitrack.Api.Models.Entities;

public class Product
{
    public Guid Id { get; set; }
    public Guid FkTenant { get; set; }
    public string Name { get; set; } = string.Empty;
    public decimal Price { get; set; }
    public decimal TaxRate { get; set; }
    public bool IsActive { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string SyncStatus { get; set; } = "synced";
}
